go.mod

```diff
@@ -0,0 +1,44 @@
+module github.com/giantswarm/ingress-exporter
+
+go 1.14
+
+require (
+	github.com/giantswarm/apiextensions v0.3.0
+	github.com/giantswarm/exporterkit v0.2.0
+	github.com/giantswarm/k8sclient v0.2.0
+	github.com/giantswarm/microendpoint v0.2.0
+	github.com/giantswarm/microerror v0.2.0
+	github.com/giantswarm/microkit v0.2.1
+	github.com/giantswarm/micrologger v0.3.1
+	github.com/giantswarm/operatorkit v0.2.0
+	github.com/prometheus/client_golang v1.3.0
+	github.com/spf13/viper v1.6.3
+	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
+	k8s.io/api v0.16.6
+	k8s.io/apimachinery v0.16.6
+	k8s.io/client-go v0.16.6
```

Comment:
[The latest aws-operator uses 0.17.2](https://github.com/giantswarm/aws-operator/blob/5da9ed7374390cdeb031dfeec85e03da8c7dfc40/go.mod#L41) so I guess we should use that here too .

Reply:
Sure

Reply:
There's a `replace` directive in the `go.mod` below which looks like it's pointing to 1.16.3. We should probably fix this so everything is using 1.16.6 for consistency.

Reply:
See https://github.com/kubernetes/client-go/commit/6c5935290e33 (from pseudo version)

Reply:
I'll make sure it's consistent and using `1.16.6` then.

Reply:
Oh, I actually have it replaced to the same version as aws-op, nevermind then.